src/common/dsp/QuadFilterUnit.cpp

```diff
@@ -691,7 +691,7 @@ __m128 TANH(__m128 in, __m128 drive)
    return _mm_max_ps(_mm_min_ps(y, y_max), y_min);
 }
 
-#if !_M_X64
+#if !_M_X64 && !defined(NEON_SSE)
```

Comment:
These SSE1 instructions aren't actually used anywhere now anyway - I should really remove them.